src/ceph.in

```diff
@@ -552,8 +552,8 @@ def ping_monitor(cluster_handle, name, timeout):
         return 1
 
     mon_id = name[len('mon.'):]
+    run_in_thread(cluster_handle.connect, timeout=timeout)
```

Comment:
If I said ping mon.a, it might work. However, that's not the name of the mon; the name is mon.mira021.  Yes, build_initial_monmap would use [mon] sections if there were any, but there aren't (and should be none on most cluster configs, IMO).  And yes, ENOENT is a result of having an initial monmap that has noname-a, noname-b, and noname-c, none of which are the real mon names.